WordPress/Classes/Utility/WPImmuTableRows.swift

```diff
@@ -8,16 +8,18 @@ struct NavigationItemRow : ImmuTableRow {
     let title: String
     let icon: UIImage?
     let action: ImmuTableAction?
+    let showsDisclosureIndicator: Bool
```

Comment:
Maybe this should allow all accessories via something like `accessoryType`. (Not sure if that is the intended use of `WPImmuTableRows`.